nixos/modules/services/cluster/k3s/default.nix

```
@@ -351,12 +351,16 @@ in
             sha256 = "0imblp0kw9vkcr7sp962jmj20fpmb3hvd3hmf4cs4x04klnq3k90";
             finalImageTag = "21.1.2-debian-11-r0";
           })
+
+          config.services.k3s.package.airgapImagesAmd64
```

question: don't you need to specify passthru?

```suggestion
          config.services.k3s.package.passthru.airgapImagesAmd64
```

---

AFAIK this makes no difference. I get the same store path when building `k3s.airgapImagesAmd64` and `k3s.passthru.airgapImagesAmd64`